researchstudio-sat/webofneeds
# Feat openimages downloadfiles

fixes #2251 -> prevent drop of filetypes that are not allowed 
fixes #2252 -> download file on click in viewer, and open images in a new tab on click

How To Test:
Create a custom need with "Files" Detail -> this detail should allow the selection and dragNdrop-Upload of any given filetype
Create an additional "Images" Detail -> this detail should only allow images in the selection and dragNdrop-Upload

After need creation, you can view the post-info of the need and see that you can click on a file to download it, and clicking on an image will open it in a new tab
